README.md

```diff
@@ -27,28 +28,25 @@ Offical exam objectives you review and understand in order to pass the test.
 
 * [CNCF Exam Curriculum repository ](https://github.com/cncf/curriculum/)
 
-- Duration : two (2) hours
-- Number of questions: 15-20 hands-on performance based tasks
-- Passing score: 67%
-- Certification validity: two (2) years
-- Prerequisite: valid CKA
-- Cost: $300 USD, One (1) year exam eligibility, with a free retake within the year.
+- **Duration** : two (2) hours
+- **Number of questions**: 15-20 hands-on performance based tasks
+- **Passing score**: 67%
+- **Certification validity**: two (2) years
+- **Prerequisite**: valid CKA
+- **Cost**: $300 USD, One (1) year exam eligibility, with a free retake within the year.
 
   *Linux Foundation offer several discounts around the year e.g. CyberMonday, Kubecon attendees among other special holidays/events*
 
-### URLs allowed in the extra single tab
-  - From Chrome or Chromium browser to open one additional tab in order to access 
-    Kubernetes Documentation: 
-    - https://kubernetes.io/docs and their subdomains
-    - https://github.com/kubernetes and their subdomains
-    - https://kubernetes.io/blog and their subdomains
-
-  This includes all available language translations of these pages (e.g. https://kubernetes.io/zh/docs)
-  - Tools:
-    - [Trivy documentation](https://github.com/aquasecurity/trivy)
-    - [Sysdig documentation](https://docs.sysdig.com)
-    - [Falco documentation](https://falco.org/docs)
-    - [App Armor documentation](https://gitlab.com/apparmor/apparmor/-/wikis/Documentation)
+## URLs allowed in the extra single tab
+From Chrome or Chromium browser to open one (1) additional tab in order to access Kubernetes Documentation: 
+- https://kubernetes.io/docs and their subdomains
```

Comment:
actually, subdomains here could be wrong as discuss for example is not allowed. my mistake